strings/sillytips.txt

```diff
@@ -11,13 +11,13 @@ Did you know that tossing the clown into a singularity will either increase or d
 Did you know that tossing the clown into the Supermatter engine can destabilize or fix it massively?
 Do not go gentle into that good night.
 FEED ME A STRAY CAT
-Flashbangs can weaken blob tiles, allowing for you and the crew to easily destroy them.
 It's fun to try and predict the round type from the tip of the round message.
 Just like real life the entropy of the game can only increase with time. If things aren't on fire yet, just wait.
 Just the tip?
 Killing the entire station isn't fun except when it is.
 Make sure you put the fresh corpse on a Nanotrasen-Brand Stasis Bed! (Patent Pending)
 Most items have names longer than "soap".
+Nanotrasen doesn't want you to know, you can print Tinfoil Hats by hacking the Autolathe at cargo, It will protect you from ...aliens, and a... plethora of mind-based magic.
```

Comment:
```suggestion
Nanotrasen doesn't want you to know, you can print Tinfoil Hats by hacking the Autolathe at cargo. It will protect you from... aliens, and a... plethora of mind-based magic.
```